src/components/html/EcApplication.html

```diff
@@ -1,131 +1,197 @@
 <div class="ec-application">
-  <a-row :gutter="24">
-    <div v-if="isLoaded">
-      <a-col :md="24" :lg="7">
-        <a-card>
-          <div class="text-center">
-            <a-avatar v-if="icon" :src="icon" :size="170"  class="mb-3 img-fluid"/>
-            <a-avatar
-              v-else
-              :size="170"
-              :style="{ background: randomColors, fontSize: '4rem' }"
-              class="mb-3 img-fluid"
-            >
-              {{ title.charAt(0) }}
-            </a-avatar>
-
-            <a-card-meta
-              :title="title"
-              class="overflow-hidden"
-            >
-              <template slot="description">
-                {{ shortDescription }}
-              </template>
-            </a-card-meta>
-          </div>
+  <div 
+    class="row d-flex justify-content-around" 
+    v-if="isLoaded"
+  >
+    <div class="col-3 card bg-white border h-100">
```

Comment:
```suggestion
    <div class="col-3 card h-100">
```

Não devemos forçar estilos para aproximar detalhes do que estava anteriormente, o anterior na verdade era o padrão do Antd e não necessariamente é o correto.
Neste caso só devemos usar o `.card`, se os cards vão ter borda ou não é o tema do admin que vai definir, sacou?

Reply:
Outro ponto aqui, em mobile esse `.col-3` não vai ficar bacana

Reply:
E outra coisa kkk
O `.card` não pode ser a mesma div do `.col` (em geral `.col` e `.row` não devem ser outro componente ao mesmo tempo), avacalha paddding.